tests/conftest.py

```diff
@@ -283,9 +293,11 @@ def fixture_fastq_dir(fixtures_dir: Path) -> Path:
 
 
 @pytest.fixture(scope="function", name="project_dir")
-def fixture_project_dir(tmpdir_factory):
+def fixture_project_dir(
+    tmpdir_factory,
+) -> Generator[Path, None, None]:
     """Path to a temporary directory where intermediate files can be stored"""
-    my_tmpdir = Path(tmpdir_factory.mktemp("data"))
+    my_tmpdir: Path = Path(tmpdir_factory.mktemp("data"))
     yield my_tmpdir
     shutil.rmtree(str(my_tmpdir))
```

Comment:
I wonder if you explicitly need to remove this afterwards. Is it not done automatically?

Reply:
I... I dont know, the fixture was there from before. I assume it should be torn down upon test completion

Reply:
No need to remove manually :P